l10n_fi_payment_reference/readme/CONFIGURE.rst

```diff
@@ -0,0 +1 @@
+Payment Reference Type can be selected from Accounting Settings.
```

Comment:
Please update to match the current implementation

Reply:
Updated